src/Mount.cpp

```diff
@@ -3039,12 +3033,15 @@ void Mount::loop()
                 _totalDECMove = _totalRAMove = 0;
 
                 // Make sure we do one last update when the steppers have stopped.
+                // TODO this should probably be throttled, 250ms won't make a difference here.
                 displayStepperPosition();
             }
         }
     }
 
     _stepperWasRunning = raStillRunning || decStillRunning;
+
+    displayStepperPositionThrottled();
```

Comment:
Why did you add this?

Reply:
It was moved from loop() in c_buttons to here. Trying to consolidate how many calls are coming from loop() into each Service/Class